impl/otbn/rsa/cond_sub_to_reg.i.vad

```diff
@@ -193,6 +193,152 @@ procedure sub_to_reg(
         init_view, iter.buff);
 }
 
+procedure sub_to_reg_copy(
+    ghost init_view: seq(uint256),
+    ghost init_iter: iter_t)
+    returns (ghost curr_view: seq(uint256))
+
+    requires
+        valid_wdr_view(wdrs, init_view, 4, NUM_WORDS);
+
+        iter_safe(init_iter, heap, x16);
+        init_iter.index == 0;
+        seq_len(init_iter.buff) == NUM_WORDS;
+        w31 == 0;
+
+    reads
+        x16; flat; heap;
+        w4; w5; w6; w7; w8; w9; w10; w11; w12; w13; w14; w15;
+
+    modifies
+        fgroups;
+        x8; x9; x10; x11; x12;
+        wdrs; w2; w3; w16; w17; w18; w19;
+        w20; w21; w22; w23; w24; w25; w26; w27; w31;
+
+    ensures
```

Comment:
For the reads only registers we can avoid stating the values stay the same. There are some other cases where `{frame: false}` is on, so we had to explicitly state the unchanged registers retain the values in this style.